src/mixins/filing-mixin.ts

```diff
@@ -160,8 +165,9 @@ export default class FilingMixin extends DateMixin {
    * Builds a Restoration filing body.
    * @returns the filing body
    */
-  buildRestorationFiling (): RestorationFilingIF {
-    const restorationFiling: RestorationFilingIF = {
+  buildRestorationFiling (restorationType: RestorationTypes = RestorationTypes.FULL): RestorationFilingIF {
+    // For security reasons, restoration.expiry and restoration.legalName should be populated by the API
```

Comment:
Thanks @severinbeauvais.  I'll remove the comment.  For the purposes of creating a draft, do we even need to submit a payload to the API?  It seems to me most (all?) the values we're submitting should be overridden by the API.

Reply:
Do you have an example of a draft in which we don't need to save info? What do you have in mind?

Reply:
Sure, for example, it looks to me like most of the restoration create payload could be generated by the API since the `identifier` is a URL parameter and all other values (except the `restoration` object) could be derived from the identifier.
- header:
  - date: <don't trust what the client tells us>
  - name: <always restoration>
- business:
  - foundingDate: <this can be looked up -- right?>
  - identifier: <get from the url parameter>
  - legalName: <this can be looked up -- right?>
  - legalType: <this can be looked up -- right?>
- restoration:
  - type: <this is the only value that the front-end needs to submit>
Sending only a payload with:

Reply:
You're mostly right and I agree with you.

(The header contains some fields from the UI: who certified, staff payment props, etc.)